webviz_config/_config_parser.py

```diff
@@ -122,6 +122,15 @@ def _call_signature(
     )
     if deprecated_plugin:
         deprecation_warnings.append(deprecated_plugin.short_message)
+        warnings.warn(
+            f"""Plugin {deprecated_plugin.class_reference} has been deprecated.
```

Comment:
```suggestion
            f"""Plugin {plugin_name} has been deprecated.
```
Is this better wrt. the end users? The `deprecated_plugin.class_reference` might look a bit technical.